src/applications/vaos/tests/appointment-list/components/UpcomingAppointmentsList.unit.spec.jsx

```diff
@@ -600,6 +600,7 @@ describe('VAOS <UpcomingAppointmentsList> V2 api', () => {
       featureToggles: {
         ...initialState.featureToggles,
         vaOnlineSchedulingVAOSServiceVAAppointments: true,
+        vaOnlineSchedulingVAOSServiceCCAppointments: true,
```

Comment:
Why was this necessary? I think these are the tests for the non-homepage refresh pages.

Reply:
Not necessary. Removing these lines.

Reply:
Actually, I added these because not having them meant getting an error when we fetched cc appointments from the v0 endpoint. So these flags were added so the tests in this block just had to fetch from v2.

Reply:
Gotcha, I was also wrong about which tests these were.